vladiate/validators.py

```diff
@@ -208,3 +210,26 @@ def validate(self, field, row={}):
     @property
     def bad(self):
         pass
+
+
+def stringify_set(a_set, max_len, max_sort_size=8192):
```

Comment:
Could we rename this to `_stringify_set` to indicate that it is not part of the public API?